### PR TITLE
TELCODOCS-1867 FRR-k8s GA

### PR DESCRIPTION
**Pull request by slovern**
[TELCODOCS-1867](https://issues.redhat.com//browse/TELCODOCS-1867) - [CNF-11914](https://issues.redhat.com//browse/CNF-11914) [GA] MetalLB - moving `FRR-k8s` from Tech Preview to GA

Version(s): 4.17

Issue: https://issues.redhat.com/browse/TELCODOCS-1867

Link to docs preview: [Configuring the integration of MetalLB and FRR-K8s ](https://80279--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/metallb/metallb-frr-k8s.html)

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by gkopels — APPROVED** [review]

File: networking/metallb/metallb-frr-k8s.adoc [hunk 1]
```diff
@@ -7,7 +7,6 @@ include::_attributes/common-attributes.adoc[]
 toc::[]
 
 :FeatureName: The `FRRConfiguration` custom resource
```
Comment (jldohmann) [inline networking/metallb/metallb-frr-k8s.adoc:9]: should this be removed as well?

**Review by jldohmann — APPROVED** [review]
LGTM, just a small question

**Comment by slovern** [comment]
Thanks @jldohmann 
That's a good point. The `FeatureName` is no longer needed once the snippet is removed. I'll update it.

**Review by skopacz1 — APPROVED** [review]
Merge review LGTM

**Pull request merged**